build-system/tasks/e2e/describes-e2e.js

```diff
@@ -479,34 +480,43 @@ class EndToEndFixture {
   /**
    * @param {!Object} env
    * @param {string} browserName
+   * @param {number} retries
```

Comment:
This may be unnecessary if the constant initializes itself with `argv.retries`. Something like:

```
const TEST_RETRIES = argv.retries || 1;
```

Reply:
I don't think it needs to be a flag because this is setup retries, not test retries.

Reply:
Also I'll need `retries` as a parameter to know when to stop the recursion.